frontend/public/components/build-pipeline.tsx

```diff
@@ -38,11 +38,12 @@ const BuildSummaryStatusIcon: React.SFC<BuildSummaryStatusIconProps> = ({ status
     </span>;
 };
 
-const BuildLogLink: React.SFC<BuildLogLinkProps> = ({ obj }) => {
+export const BuildLogLink: React.SFC<BuildLogLinkProps> = ({ obj, linkClass = '' }) => {
   const link = getJenkinsLogURL(obj);
-  return link && <div className="build-pipeline__link">
-    <a href={link} className="build-pipeline__log-link" target="_blank" rel="noopener noreferrer">View Log</a>
-  </div>;
+  return link ?
+    <div className="build-pipeline__link">
+      <a href={link} className={linkClass} target="_blank" rel="noopener noreferrer">View Log</a>
+    </div> : <LoadingInline/>;
```

Comment:
Just following up here.

I still don't think we add the animation. The loading animation could literally be there for hours if you run pipelines serially. (New pipelines won't start until all others queued ahead of it have completed.)

Reply:
Need to prettify this :)
```js
  return link
    ? <div className="build-pipeline__link">
       <a href={link} className={linkClass} target="_blank" rel="noopener noreferrer">View Log</a>
    </div>
    : <LoadingInline/>;
```

Reply:
I get nervous with this style due to the possibility automatic semicolon insertion, but maybe it's not a problem here. I forget all the rules.

It also makes `<div>` and `<a>` line up even though `<a>` is nested because of what's enforced by the linter :/

Reply:
Thought that its the style we [use](https://github.com/openshift/console/blob/master/frontend/public/components/cloud-services/clusterserviceversion-resource.tsx#L212-L216) :)

Reply:
@spadgett the auto semicolor insertion will only happen if the `return` is on it's own line:

```JavaScript
return    // doomed
  something ? true : false;
```
We definitely don't do that.

Reply:
I discussed the loading animation with @cshinn and we agree that it doesn't make sense to show the loading animation on the Overview tab since there are other indications of the pending status of the build. On the other hand, it might make sense on the Logs tab. Since the user is specifically looking for logs on that page, they might gloss over the empty state message there and assume there is a problem. There isn't any other information on the Logs tab that indicate a forthcoming link to the Jenkins log.